assets/src/routes/routes.js

```diff
@@ -1,41 +1,50 @@
 import { Grade } from '@material-ui/icons'
+import { viewHelpURLs } from '../globals'
```

Comment:
This wasn't done anywhere else, so I'm not sure if it is considered bad or if its OK.

Reply:
IMO This should be ok since you are just pulling what is needed rather bring all the stuff

Reply:
This should be ok.